src/components/Trade/hooks/useTradeRoutes/useTradeRoutes.ts

```diff
@@ -23,7 +24,9 @@ export const useTradeRoutes = (
   const { updateQuote, getDefaultPair, swapperManager } = useSwapper()
   const buyTradeAsset = getValues('buyAsset')
   const sellTradeAsset = getValues('sellAsset')
-  const feeAssetId = chainIdToFeeAssetId(sellTradeAsset?.asset?.chainId ?? 'eip155:1')
+  const feeAssetId = getChainAdapters()
+    .get(sellTradeAsset?.asset?.chainId ?? 'eip155:1')
```

Comment:
```suggestion
    .get(sellTradeAsset?.asset?.chainId ?? ethChainId)
```

Reply:
Updated in fdf566cd4117707ce5dcf39ec6bca251abd90be8